cypress/integration/footer-feature.spec.js

```diff
@@ -0,0 +1,236 @@
+                .contains('Directions')
+                .should('have.attr', 'href', "https://www.google.com/maps/dir/''/320+York+Rd,+Towson,+MD+21204");
+        });
+
+        it('should contain a link that navigates to the Ask A Librarian page', () => {
+            cy
+                .get(`${constants.selectors.calloutDivs}`)
+                .contains('Ask a Librarian')
+                .should('have.attr', 'href', "/services/ask-a-librarian.html");
+        });
+
+        it('should contain a link that navigates to the Digital Library page', () => {
+            cy
+                .get(`${constants.selectors.calloutDivs}`)
+                .contains('Explore Digital Materials')
+                .should('have.attr', 'href', "/books-and-more/downloadables.html");
+        });
+
+        it('should contain a link that navigates to the Job Opportunities page', () => {
+            cy
+                .get(`${constants.selectors.calloutDivs}`)
+                .contains('Search Jobs')
+                .should('have.attr', 'href', "/about-us/job-opportunities.html");
+        });
+
+    });
+
+    describe('Bottom Nav', () => {
+        // These tests currently take a long time due to cy.visit()
+        // waiting for onLoad to complete. Cypress will be adding the ability
+        // to complete on DOMContentLoaded which may speed up tests.
+        // Open issue : https://github.com/cypress-io/cypress/issues/440
+
+        beforeEach(() => {
+            cy.visit(constants.rootUrl);
+        });
+
+        it('should conatin a text link containing "User Terms" that points to the policies page', () => {
+            cy
+                .contains('a', 'User Terms')
+                .should('have.attr', 'href', '/about-us/policies.html#user-terms')
+                .click();
+
+            cy
+                .get('a[name=user-terms]')
+                .should('exist');
+        });
+
+        it('should conatin a text link containing "Privacy Policy" that points to the policies page', () => {
+            cy
+                .contains('a', 'Privacy Policy')
+                .should('have.attr', 'href', '/about-us/policies.html#privacy-policy')
+                .click();
+
+            cy
+                .get('a[name=privacy-policy]')
+                .should('exist');
+        });
+
+        it('should conatin a text link containing "Accessibility" that points to the policies page', () => {
+            cy
+                .contains('a', 'Accessibility')
+                .should('have.attr', 'href', '/about-us/policies.html#accessibility-statement')
+                .click();
+
+            cy
+                .get('a[name=accessibility-statement]')
+                .should('exist');
+        });
+
+    });
+
+    describe('Adjust Contrast', () => {
+        // Backup the clear function.
+        // This is going to be set to null below...
+        const clear = Cypress.LocalStorage.clear;
```

Comment:
nice.

Reply:
you might want to call it clearLocalStorage in the future, but don't think this is killer right now, since it's so isolated.